mesonbuild/modules/gnome.py

```diff
@@ -1138,7 +1138,10 @@ def generate_vapi(self, state, args, kwargs):
         build_dir = os.path.join(state.environment.get_build_dir(), state.subdir)
         source_dir = os.path.join(state.environment.get_source_dir(), state.subdir)
         pkg_cmd, vapi_depends, vapi_packages, vapi_includes = self._extract_vapi_packages(state, kwargs)
-        cmd = [find_program('vapigen', 'Vaapi')]
+        if 'VAPIGEN' in os.environ:
+            cmd = [find_program(os.environ['VAPIGEN'], 'Vaapi')]
+        else:
+            cmd = [find_program('vapigen', 'Vaapi')]
```

Comment:
What is 'Vaapi'? It looks from `find_program` it is a target name used in error report. From `mesonbuild/modules/__init__.py`:

```
def find_program(program_name, target_name):
    if program_name in _found_programs:
        return _found_programs[program_name]
    program = dependencies.ExternalProgram(program_name)
    if not program.found():
        m = "Target {!r} can't be generated as {!r} could not be found"
        raise MesonException(m.format(target_name, program_name))
    _found_programs[program_name] = program
return program
```

Is this just a typo? If so then it is probably better as `VAPI`.